src/targets/github.ts

```diff
@@ -379,11 +379,12 @@ export class GitHubTarget extends BaseTarget {
       revision,
       changelog
     );
-    await Promise.all(
-      localArtifacts.map(({ path, mimeType }) =>
-        this.uploadAsset(draftRelease, path, mimeType)
-      )
-    );
+
+    // We don't want to do this in parallel but in serial, because the GitHub
+    // artifacts endpoint has a tendency to randomly timeout on multiple requests.
+    for (const { path, mimeType } of localArtifacts) {
+      await this.uploadAsset(draftRelease, path, mimeType);
```

Comment:
Do we know how much this will potentially slow down stuff by?

Reply:
I don't have any specific number. However, we don't publish any repositories that contain more than 10-20 artifacts to GitHub afaik.